docs/contribute/index.md

```diff
@@ -1 +1,30 @@
-# Hello
+# Hello contributor!
+
+In this section, we're gone to explain how to contribute to this project and some guides.
```

Comment:
In this section, we're going to explain how to contribute to this project and some guides.